lib/node.js

```diff
@@ -345,7 +345,13 @@ Node.prototype.getInfo = function()
 
 	try {
 		this.info.coinbase = web3.eth.coinbase;
```

Comment:
Maybe `this.info.node = web3.version.node;` should be here?

Reply:
The problem here seems to be that calling `coinbase` throws an exception so `this.info.node` is not actually evaluated at all. Now this exception should be handled, so the rest of the code works

Reply:
I get what you mean now, sorry :)